x-pack/plugins/data_usage/public/plugin.ts

```diff
@@ -24,30 +29,46 @@ export class DataUsagePlugin
       DataUsageStartDependencies
     >
 {
+  private config: DataUsagePublicConfigType;
+  private experimentalFeatures: ExperimentalFeatures;
+
+  constructor(private readonly initializerContext: PluginInitializerContext) {
+    this.config = this.initializerContext.config.get<DataUsagePublicConfigType>();
+    this.experimentalFeatures = {} as ExperimentalFeatures;
+  }
+
   public setup(
     core: CoreSetup<DataUsageStartDependencies, DataUsagePublicStart>,
     plugins: DataUsageSetupDependencies
   ): DataUsagePublicSetup {
     const { management } = plugins;
-    management.sections.section.data.registerApp({
-      id: PLUGIN_ID,
-      title: PLUGIN_NAME,
-      order: 6,
-      keywords: ['data usage', 'usage'],
-      async mount(params: ManagementAppMountParams) {
-        const [{ renderApp }, [coreStart, pluginsStartDeps, pluginStart]] = await Promise.all([
-          import('./application'),
-          core.getStartServices(),
-        ]);
-
-        return renderApp(coreStart, pluginsStartDeps, pluginStart, params);
-      },
-    });
 
+    this.experimentalFeatures = parseExperimentalConfigValue(
+      this.config.enableExperimental
+    ).features;
+
+    const experimentalFeatures = this.experimentalFeatures;
+
+    if (!experimentalFeatures.dataUsageDisabled) {
```

Comment:
```suggestion
    if (experimentalFeatures.dataUsageEnabled) {
```

Reply:
The logic introduced by `xpack.dataUsage.enableExperimental: ['dataUsageDisabled']` is about hiding data usage functionality for Serverless users after we enabled the plugin. This will simplify our testing for a different project types and environments (AutoOps API on prod is not ready yet, and we should have the ability to test it).